package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hubot-aws-cfpb",
-  "version": "1.0.8",
+  "version": "1.0.9",
```

Comment:
Bump the minor version (`1.0.8` > `1.1.0`) because this change adds backward-compatible features.